modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_1_5/test/PaginatedDDMStructureLayoutCTUpgradeProcessTest.java

```diff
@@ -6,16 +6,16 @@
 package com.liferay.dynamic.data.mapping.upgrade.v5_1_5.test;
 
 import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
-import com.liferay.dynamic.data.mapping.upgrade.BaseDDMStructureLayoutCTUpgradeProcessTestCase;
+import com.liferay.dynamic.data.mapping.upgrade.v4_3_3.test.DDMStructureLayoutCTUpgradeProcessTest;
 
 import org.junit.runner.RunWith;
 
 /**
  * @author Paulo Albuquerque
  */
 @RunWith(Arquillian.class)
-public class DDMStructureLayoutUpgradeProcessTest
-	extends BaseDDMStructureLayoutCTUpgradeProcessTestCase {
+public class PaginatedDDMStructureLayoutCTUpgradeProcessTest
+	extends DDMStructureLayoutCTUpgradeProcessTest {
```

Comment:
What do you think @marcelabc ? 

I think we can keep the same same:

```
DDMStructureLayoutCTUpgradeProcessTest extends com.liferay.dynamic.data.mapping.upgrade.v4_3_3.test.DDMStructureLayoutCTUpgradeProcessTest
```

similar to what we have [here](https://github.com/liferay/liferay-portal/blob/6d3356ad85b36dd8690290bb54e2caf53b3c79d2/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_4_3/PollsPortletIdToDDMPortletIdUpgradeProcess.java#L11-L13).

Reply:
I agree with @carolmariaabb.

Setting a prefix can be tricky because there is no rule for the following examples. We would have to understand the specific upgrade process to think about a prefix that would make sense for that upgrade. There is no pattern on that :confused:

Reply:
What do you think @MarinhoFeliphe?

Reply:
Makes sense to me! [see](https://github.com/liferay-forms/liferay-portal/pull/2587/commits/4bcd5e81fd06b92eaef25833648a37a62992fe50)

Thank you!